backend/twitterbot/management/commands/tweet_top_contribors.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management.base import BaseCommand
+
+from twitterbot.tournesolbot import tweet_top_contributor_graph
+
+
+class Command(BaseCommand):
+    help = "Runs Tournesol Twitter Bot to tweet Top contributor figure"
+
+    def add_arguments(self, parser):
+
+        parser.add_argument(
+            "-n",
+            "--bot-name",
+            type=str,
+            help="Name of the bot '@TournesolBot' or '@TournesolBotFR'",
+        )
```

Comment:
Would make sense to make this argument required

```suggestion
        parser.add_argument(
            "-n",
            "--bot-name",
            type=str,
            required=True,
            help="Name of the bot '@TournesolBot' or '@TournesolBotFR'",
        )
```